src/components/Editor/components/SaveButton.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+/**-------Props--------
+ * runCode: function to be called when button is pressed
+ */
+const RunButton = props => (
```

Comment:
Bc we're making a common Button later, no need to change, but in the future remember to update the component name!